go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/patrickmn/go-cache v2.1.0+incompatible
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.7.1
-	github.com/storageos/go-api/v2 v2.3.1-0.20210211170707-7520f2fe0974
+	github.com/storageos/go-api/v2 v2.3.1-0.20210420150320-a30cf41359d2
```

Comment:
I had trouble setting to `v2.4.0` so used the commit ref for the tag instead.  I might have something cached as I'd imported from a branch named `v2.4.0` before I tagged.